projects/components/src/quick-search/quick-search.component.html

```diff
@@ -31,6 +31,8 @@
             >
                 <button
                     class="dropdown-toggle btn btn-link"
+                    id="{{ filter.id }}"
+                    [attr.aria-labelledby]="'filter-label ' + filter.id"
```

Comment:
I don't understand the change, the button already has text inside of it which would be read. You created a `<p>` with the ID of "filter-label" but added `[attr.aria-labelledby]="'filter-label ' + filter.id"`

It seems like it will read "filter by 'filter name'" but it would seem simpler to just add an aria-label on each checkbox

Reply:
Yes, there is another reason to do it with an `aria-label`. This string concatenation is problematic for translation because different languages could have words in different order.
The `<p>` tag was before, so will leave it as it was.